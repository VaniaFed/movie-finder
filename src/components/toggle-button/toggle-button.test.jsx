import renderer from 'react-test-renderer';
import { ToggleButton } from './index';

describe('<ToggleButton />', () => {
    const tree = renderer
        .create(
            <ToggleButton
                label="Some label"
                checked
                background
                name="some_name"
                onClick={f => f}
            />
        )
        .toJSON();
    it('should be rendered', () => {
        expect(tree).toMatchSnapshot();
    });
});
