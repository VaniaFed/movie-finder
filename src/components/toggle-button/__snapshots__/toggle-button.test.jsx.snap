// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToggleButton /> should be rendered 1`] = `
<label
  className="toggle-button"
  onClick={[Function]}
>
  <input
    className="toggle-button__input toggle-button__input--checked"
    defaultChecked={true}
    name="some_name"
    type="radio"
  />
  <span
    className="toggle-button__text toggle-button__text--checked"
  >
    Some label
  </span>
</label>
`;
